client/state/notices/README.md

```diff
@@ -0,0 +1,53 @@
+Notices
+===========
+
+A module for managing global notices.
+
+## Create a notice
+
+Provides some helper functions to render a notice on the UI. Types of notices are `error`, `success`.
+
+### How to use?
+
+#### Connect to Redux
+First, the component that wants to send notices must be connected to Redux.
+
+```javascript
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { successNotice, errorNotice } from 'state/notices/actions';
+
+...
+
+export default connect(
+	null,
+	dispatch => bindActionCreators( { successNotice, errorNotice }, dispatch )
+)( Component );
+```
+
+#### Uses
+
+Now you can use notices like so:
+
+```javascript
+
+this.props.successNotice(
+    'Settings saved successfully!', {
+    	duration: 4000
+} );
+
+```
+
+
+#### The available methods are:
+
+* `notices.success()`
+* `notices.error()`
```

Comment:
These aren't the names of the methods... they're `successNotice` and `errorNotice`... and there's also `removeNotice`. They also would not typically be called as attributes on an object like this now that we're using destructuring most everywhere.

Sidenote, the `thunks` are interesting. Not quite actions, not quite action creators. I still feel like a noob in our redux efforts... is this approach common? Why is `removeNotice` a straight action creator then?